static/global.js

```diff
@@ -12,3 +12,65 @@
   r.src = t + h._hjSettings.hjid + j + h._hjSettings.hjsv;
   a.appendChild(r);
 })(window, document, "https://static.hotjar.com/c/hotjar-", ".js?sv=");
+
+// Segment
```

Comment:
Segment is already loaded into the site by a post-deploy script on Netlify - so this part is redundant

Reply:
Removed